doc/nrf/known_issues.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
is that normal?

Reply:
Yes, we don't want the known issues to show up in the table of contents for a release or dev tag, only in the 'latest' documentation from the main branch.